builder.sh

```diff
@@ -894,19 +895,6 @@ while [[ $# -gt 0 ]]; do
             extract_machine_build "$(echo "$2" | cut -d '=' -f 2)"
             shift
             ;;
-        --with-codenotary)
```

Comment:
Instead of removing these completely should we create a warning if they are used?

Reply:
Oh, he got a warning if he using not used parameters

Reply:
Ah, right <https://github.com/home-assistant/builder/pull/112/files#L911> :+1: